src/Tags/Video.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace Spatie\Sitemap\Tags;
+
+class Video
+{
+    public const PLATFORM_WEB    = 'web';
+    public const PLATFORM_MOBILE = 'mobile';
+    public const PLATFORM_TV     = 'tv';
+
+    public const BOOL_FALSE      = "no";
```

Comment:
It isn't clear by the name where there are being used. Rename them to the parameter where they are used.

Reply:
These constants can be used in optional tags. Although they are not used internally by the package itself, they're defined to make the developer's work easier. 

For example, if the users want to create the following tag:

```
<url>
 <loc>https://www.example.com/videos/some_video_landing_page.html</loc>
 <video:video>
   <video:thumbnail_loc>https://www.example.com/thumbs/345.jpg</video:thumbnail_loc>
   <video:title>Grilling steaks for winter</video:title>
   ....
   <video:platform relationship="allow">web</video:platform>
 </video:video>
</url>
```

They can use the package's constants instead of writing them as a plain string:

```
$allow = ["platform" => Video::PLATFORM_WEB];
$sitemap = Sitemap::create()
    ->add(
        Url::create("https://example.com")
            ->addVideo("https://example.com/image.jpg", ... , $options, $allow, $deny)
    );
```

I can use an `OPTION_` prefix for these consts.

Reply:
Yeah,  `OPTION_` might be better 👍